tests/PySys/software_management_end_to_end/dummy_plugin_configuration/Info.md

```diff
@@ -19,12 +19,12 @@ When executed via the software management agent, this path will be used:
 
 ## Emulation of fruits
 
-The current trick is just to use a static configuration to emulate a funcional
-plugin that lets us install and deinstall fruits.
+The current trick is just to use a static configuration to emulate a functional
+plugin that lets us install and de-install fruits.
 Right now, the apt plugin only cares about the returned text on the list command.
 So we just insert a static list and base tests on that.
 
-Note that even when a package is deinstalled it will still appear in the list
+Note that even when a package is de-installed it will still appear in the list
 as the response is static.
```

Comment:
I see that as an important issue to be fixed in the dummy plugin.

* A plugin implemented for testing purpose *must* follow the requirements.
* Using a static file to mimic the repository is fine but not to manage what is installed or not.

Reply:
The dummy plugin that we currently have responds with a static response loaded from a text-config file. Currently, it has no other state. We could update the config file to change the response (or rewrite the plugin).